### DISCUSSION TITLE
disable dialog closing when click outside of dialog

### DISCUSSION THREAD

**Discussion by oliverkan** [tailwindlabs/headlessui#1700] [Help]
Hi, I am trying to find a way to make dialog persistent on screen. I mean I need to disable the closing feature of dialog when the user clicks somewhere outside of the dialog. I tried the `static` prop but it didn't help. I read the solution alternatives in this discussion. They mention about a tailwind class for pointer events but in new version Dialog.Overlay is deprecated. And also I don't want to loose `esc` key functionality if I have an empty `onclose` function. Is there any new, better way to solve this problem or should I have an empty onClose function and implement `esc` keydown event listener?

Thanks and regards.

**Comment by vFire** [comment]
Same thing...
